app/scripts/router.js

```diff
@@ -83,7 +83,8 @@ function (
         router: this,
         user: this.user,
         window: this.window,
-        screenName: this.fragmentToScreenName(Backbone.history.fragment)
+        screenName: this.fragmentToScreenName(Backbone.history.fragment),
+        model: this.formModel
```

Comment:
Is the naming of `model` intended to be generic? I.e. we'd overload `view.model` with more attributes rather than having `view.formModel`, and other `view.*Model`s?